packages/portfolio-api/docs/ymax-machine.yaml

```diff
@@ -1,130 +1,933 @@
+        description: GMP contract call to depositForBurn on EVM wallet
+        meta:
+          row: Agoric Chain
+          observedFrom:
+            - FlowStatus state=run with how=CCTP
+          txType: CCTP_TO_AGORIC
+          protocol: Circle CCTP + Axelar GMP
+        on:
+          gmp.submitted:
+            target: gmp_pending
+            description: GMP message sent via Axelar
+          gmp.failed:
+            target: failed
+            description: GMP submission failed
+      gmp_pending:
+        description: GMP message in transit to EVM chain via Axelar
+        meta:
+          row: Axelar Network
+          observedFrom:
+            - Axelar GMP explorer (axelarscan.io)
+          observabilityTodo: "TODO: Poll Axelar API for GMP message status"
+          protocol: Axelar GMP
+        on:
+          gmp.executed:
+            target: burn_pending
+            description: Axelar delivered and executed GMP payload on EVM
+          gmp.failed:
+            target: gmp_failed
+            description: GMP message delivery or execution failed
+      burn_pending:
+        description: EVM wallet executing depositForBurn
+        meta:
+          row: EVM Chain
+          observedFrom:
+            - EVM tx history for smart wallet
+          protocol: Circle CCTP
+        on:
+          burn.confirmed:
+            target: attestation_pending
+            description: USDC burned on EVM; waiting for attestation
+          burn.failed:
+            target: failed
+            description: Burn transaction reverted on EVM
+      attestation_pending:
+        description: Waiting for Circle attestation
+        meta:
+          row: Circle Attestation
+          observedFrom:
+            - Circle Attestation API
+          protocol: Circle CCTP
+          expectedSlaMs: 60000
+        on:
+          attestation.received:
+            target: noble_mint_pending
+            description: Attestation ready; mint can proceed on Noble
+          attestation.timeout:
+            target: attestation_timeout
+            description: Attestation timed out
+      noble_mint_pending:
+        description: USDC being minted on Noble to NFA (Noble Forwarding Account)
+        meta:
+          row: Noble Chain
+          observedFrom:
+            - Noble tx history showing receiveMessage
+          observabilityTodo: "TODO: Monitor Noble for CCTP receiveMessage events"
+          protocol: Circle CCTP
+        on:
+          mint.confirmed:
+            target: ibc_forward_pending
+            description: USDC minted on Noble NFA
+          mint.failed:
+            target: failed
+            description: Noble mint failed
+      ibc_forward_pending:
+        description: NFA auto-forwarding USDC to Agoric via IBC
+        meta:
+          row: IBC Relayer
+          observedFrom:
+            - Noble tx history showing auto-forward
+            - Relayer logs
+          protocol: IBC
+        on:
+          forward.complete:
```

Comment:
forwarding is another IBC transfer susceptible to relayer availability and such.